src/test/scala/nl.knaw.dans.easy.multideposit/actions/AddBagToDepositSpec.scala

```diff
@@ -129,6 +129,26 @@ class AddBagToDepositSpec extends TestSupportFixture with BeforeAndAfterEach {
     bagInfo.read() should include("Created")
   }
 
+  it should "contain the Is-Version-Of in the bag-info.txt if the Option[BaseUUID] is Some " in {
+    action.addBagToDeposit(depositId, date, base) shouldBe a[Success[_]]
+
+    val bagInfo = stagingBagDir(depositId).resolve("bag-info.txt")
+    bagInfo.toFile should exist
+
+    bagInfo.read() should include("Is-Version-Of: " + base.get)
+  }
+
+  it should "not contain the Is-Version-Of in the bag-info.txt if the Option[BaseUUID] is None " in {
+    base = None
```

Comment:
no need to make `base` a variable rather than a value. Just plug in `None` to `action.addBagToDeposit(depositId, date, None)` instead.